src/main/java/org/folio/orders/utils/PoLineCommonUtil.java

```diff
@@ -191,6 +203,14 @@ public static Map<String, List<Location>> groupLocationsByHoldingId(CompositePoL
       .collect(Collectors.groupingBy(Location::getHoldingId));
   }
 
+  public static List<String> getLocationTenants(CompositePoLine poLine) {
```

Comment:
Propose to rename to getTenantsFromPoLineLocations or other you can come up with. Current nime like returining Pair locations and tenants

Reply:
The method name should be reading like `getOrderLinePieces`: OrderLine is an object and it contains pieces. Same way Location is an object and it contains tenant. WDYT?

Reply:
Maybe `getTenantsFromLocations` as `CompositePoLine` is a method parameter?

Reply:
Your explanation make sense, you can leave as it was or change to other name specific comments. When invoking method considering it is static method, reading getLocationTenants kind of strange, so getPoLinLocationTenants make descriptive, it is suggest that it will get poLine object. It is up to you. Both version seems good to me.